apps/system/js/app_text_selection_dialog.js

```diff
@@ -9,6 +9,11 @@
   // states which are shared by apps should be put in this class.
   var AppTextSelectionDialogGlobalStates = function() {
     this._hasCutOrCopiedTimeoutId = null;
+    this._isPrefOn = true;
+    this._appTSDArray = [];
```

Comment:
I am very sorry for giving you the wrong information, but this should be updated with a `Set()`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set